pages/index.tsx

```diff
@@ -20,14 +20,20 @@ export default function Home() {
         description="Vrei să înveți FrontEnd? Aici ai parte de tutoriale gratuite și o comunitate de developeri care te vor ajuta să devii mai bun."
       />
       <>
-        {/* <LandingAdBanner /> */}
         <LandingHero />
+        <LandingStats
```

Comment:
Let's move at the end of the home page.

Let's also add it to the events pages (at the end again)